src/BUILD.bazel

```diff
@@ -0,0 +1,35 @@
+package(default_visibility = ["//visibility:public"])
+
+# TS compile
+load("@npm_bazel_typescript//:index.bzl", "ts_library")
+
+# bazel build //src:app
+ts_library(
+    name = "app",
+    srcs = glob(
+        include = ["**/*.ts"],
+        exclude = ["**/*spec.ts"],
+    ),
+    deps = [
+        "@npm//rxjs",
+        "@npm//@nestjs",
+        "@npm//@types",
+        "@npm//tslib",
```

Comment:
Please also add `"importHelpers": true,` to the `compilerOptions` tsconfig.json section, otherwise using `tslib` dependency makes no sense in this context